test.properties

```diff
@@ -560,6 +560,9 @@
 ##
 ## Gradle Plugins
 ##
+    
+    gradle.plugins.test.excludes.globs[gradle-plugins-workspace]=\
+        StartDockerContainer
```

Comment:
@kyle-miho I tried inverting this and running `gw gradleTest -Dgradle.test.enabled=true -PtestIncludes="StartDockerContainer"` in `gradle-plugins-workspace`, and no tests were run. I think what you want instead is `*StartDockerContainerSpec*`. `gradleTest` appends `Spec` to every generated test class, so this can be a reliable way to target test cases that prefix other test cases :)